src/test/java/com/github/reducktion/socrates/validator/GermanyIdValidatorTest.java

```diff
@@ -0,0 +1,75 @@
+package com.github.reducktion.socrates.validator;
+
+import static org.hamcrest.CoreMatchers.is;
+import static org.hamcrest.MatcherAssert.assertThat;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.ValueSource;
+
+class GermanyIdValidatorTest {
+    private GermanyIdValidator germanyIdValidator;
+
+    @BeforeEach
+    void setup() {
+        germanyIdValidator = new GermanyIdValidator();
+    }
+
+    /**
+     * These valid numbers are taken from the official document,
+     * mentioned in the JavaDoc of $GermanyIdValidator#validate.
+     *
+     * @param validId an ID to test
+     */
+    @ParameterizedTest
+    @ValueSource(strings = {
+        "86095742719",
+        "47036892816",
+        "65929970489",
+        "57549285017",
+        "25768131411"
+    })
+    void these_valid_numbers_should_pass_the_test(final String validId) {
```

Comment:
Please, try to follow our test naming convention: `<method name>_should<expected behavior>_when<state under test>`.

Reply:
Renamed, following your suggestion.